src/xtgeo/surface/regular_surface.py

```diff
@@ -2415,6 +2433,87 @@ def slice_cube_window(
             algorithm=algorithm,
         )
 
+    def attributes_from_cube_window(
```

Comment:
Question to reviewers: Would this function "feel" better if moved to Cube class?

`dict_with_attr_maps = cube.attributes_between surfaces(surf1, surf2, ...)`

Reply:
I think it could be worth it if it resolves the circular dependency issue alone. It does feel more cube oriented, definitely

Reply:
I think I will go for that move!